### PR TITLE
Mapped printers

### PR DESCRIPTION
**Pull request by jochemin**
Add "Mapped Printers" scanner, it shows computer mapped printers and if it's the default one.

### CODE REVIEW

**Comment by ColbyBouma** [comment]
Inventory already scans for printers. Is your goal to show the default printer?

**Comment by jochemin** [comment]
It scans local printers. The scanner returns mapped printers.

**Comment by ColbyBouma** [comment]
When I tested #29, it returned the same printers as Inventory's Printer scanner. Your script uses the same WMI class, `Win32_Printer`.

Local, Port, and Share Name are hidden by default.
image

**Comment by jochemin** [comment]
Just tested. In the default inventory it doesn't show network printers
image
With the scanner you have that info and the default printer.
image

**Comment by ColbyBouma** [comment]
Oops, I thought it did. Good to know :)

File: PowerShell Scanners/Mapped Printers/Mapped Printers.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
+$Printers = Get-WMIObject Win32_Printer | where{$_.Name -like "*\\*"}
```
Comment (ColbyBouma) [inline PowerShell Scanners/Mapped Printers/Mapped Printers.ps1]: I believe this script can be simplified quite a bit.

```PowerShell
Get-CimInstance Win32_Printer | Where-Object Name -Like "\\*" | Select-Object ShareName, Name, Default
```

Comment (AndrewPla) [reply PowerShell Scanners/Mapped Printers/Mapped Printers.ps1]: Colby, you are correct. I have applied your suggestion and gave it a test and it works great.

**Review by AndrewPla — APPROVED** [review]
looks good and the code returns the appropriate information. Cool scanner

**Pull request merged**